install-script/install.sh

```diff
@@ -1196,50 +1197,15 @@ function add_extension_to_config() {
 
 # write sumologic extension to user configuration file
 function write_sumologic_extension() {
-    local file
-    readonly file="${1}"
-
-    local indentation
-    readonly indentation="${2}"
-
-    if sed -e '/^extensions/,/^[a-z]/!d' "${file}" | grep -qE '^\s+(sumologic|sumologic\/.*):\s*$'; then
-        return
-    fi
-
-    # add sumologic extension on the top of the extensions
-    sed -i.bak -e "s/extensions:/extensions:\\
-${indentation}sumologic:/" "${file}"
+    otelcol-config --write-kv '.extensions.sumologic = {}'
 }
```

Comment:
This is no longer necessary, otelcol-config will handle initialization of the configuration file